.changeset/eighty-taxis-wait.md

```diff
@@ -0,0 +1,16 @@
+---
+"astro": minor
```

Comment:
Since actions are experimental, I think we can flag this as `patch`. But I leave the decision to you